lib/good_job/job.rb

```diff
@@ -51,6 +106,13 @@ def self.queue_parser(string)
       end
     end)
 
+    # Finds the next eligible Job, acquire an advisory lock related to it, and
+    # executes the job.
+    # @return [Array<GoodJob::Job, Object, Exception>, nil]
+    #   If a job was executed, returns an array with the {Job} record, the
+    #   return value for the job's +#perform+ method, and the exception the job
+    #   raised, if any (if the job raised, then the second array entry will be
+    #   +nil+). If there were no jobs to execute, returns +nil+.
     def self.perform_with_advisory_lock
```

Comment:
In the README and in other spots, we used the language of “executing” jobs. This made me wonder if we should have said “perform” everywhere instead. Especially since it is the ActiveJob language.